pkg/controller/backup_controller.go

```diff
@@ -446,9 +452,18 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 		return err
 	}
 
-	var errs []error
+	exists, err := backupStore.BackupExists(backup.StorageLocation.Spec.StorageType.ObjectStorage.Bucket, backup.Name)
+	if exists || err != nil {
+		backup.Status.Phase = velerov1api.BackupPhaseFailed
+		backup.Status.CompletionTimestamp.Time = c.clock.Now()
+		if err != nil {
+			return errors.Errorf("Error checking if backup already exists in object storage: %v", err)
```

Comment:
`errors.Wrapf` is better than `errors.Errorf` here because it retains the inner error + stack trace